modules/custom/openy_activity_finder/css/activity_finder.css

```diff
@@ -791,6 +792,41 @@ and (-webkit-min-device-pixel-ratio: 2) {
   padding: 0 5px;
 }
 
+.af-schedules-empty_results .search-criteria{
+  margin: 15px auto;
+}
+
+.af-schedules-empty_results .mb {
+  margin: 20px 0 40px;
+}
+
+.af-schedules-empty_results .mt {
+  margin-top: 40px;
+}
+
+.schedule-dashboard__modal--body .modal-info-sidebar .more_results a,
+.schedule-dashboard__modal--body .modal-info-sidebar .location_name a {
+  font-size: 12px;
+}
+
+
+.schedules-data__row--alt .spots-availability {
+  border-radius: 3px;
+}
+
+@media (min-width: 768px){
+  .landing-content {
```

Comment:
Possible regressions, it's very common css rule, use something specific or find workaround